source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -0,0 +1,300 @@
+  .. _kubernetes:
+
+Run Kubernetes\* on Clear Linux
+###############################
+
+This tutorial describes how to install, configure, and run the 
+`Kubernetes system`_ on |CL-ATTR| using either the Kubernetes
+runtime interface *CRI-O* and *kata-runtime* or *Docker\** and *runc*.
```

Comment:
Let's simplify this, since it is the first sentence and overwhelming to a first time reader. 

> using either the Kubernetes runtime interface *CRI-O* and *kata-runtime* or *Docker\** and *runc*.

 to something like

> using different container engines and runtimes. 

and dissect the whole kata vs runc things in the paragraphs below.